t/db/sample_data/mlb_list.csv

```diff
@@ -0,0 +1,51 @@
+HR,First_Name,Last_Name,Username,Email,Section
```

Comment:
What is the HR column?

Reply:
It is the number of home runs. I have been using it for the student id. 

I'm thinking of making this very robust that you can load in any csv file and handle it.  The "header rows" toggle guesses what the columns are, but you can change those to whatever.

Reply:
If I toggle the header row on before loading the csv, the file is not parsed as though it has a header row. If I then turn the header row off, the data vanishes. I have to load the file without header row on, and once the data is loaded, the header row will be parsed correctly.

Hiding the header row toggle until the data is loaded would solve this issue, and simplify the import workflow.

Reply:
I disabled the header row toggle until the file was loaded.